plugins/juju/juju.plugin.zsh

```diff
@@ -0,0 +1,134 @@
+# ---------------------------------------------------------- #
+# Aliases and functions for juju (https://juju.is)           #
+# ---------------------------------------------------------- #
+
+# Load TAB completions
+_load_tab_completions() {
+  autoload -U +X compinit && compinit
+  autoload -U +X bashcompinit && bashcompinit
+  local jcomp="$(pkg-config --variable=completionsdir bash-completion)/juju"
+  [ -f "$jcomp" ] && source "$jcomp"
+}
+_load_tab_completions
+
+# ---------------------------------------------------------- #
+# Aliases (in alphabetic order)                              #
+#                                                            #
+# Generally,                                                 #
+#   - `!` means --force --no-wait -y                         #
+#   - `ds` suffix means --destroy-storage                    #
+# ---------------------------------------------------------- #
+alias jam='juju add-model --config logging-config="<root>=WARNING; unit=DEBUG"\
+  --config update-status-hook-interval="60m"'
```

Comment:
This adds a weird newline character in the middle of the alias, try it this way:
```suggestion
alias jam="juju add-model --config logging-config=\"<root>=WARNING; unit=DEBUG\"\
 --config update-status-hook-interval=\"60m\""
```